runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -50,6 +50,11 @@
   Long getBatchIntervalMillis();
   void setBatchIntervalMillis(Long batchInterval);
 
+  @Description("Default storage level")
```

Comment:
Maybe add a comment in the description that it's only applies to batch ? I agree that it doesn't have to be in the option name, but the description can be as informative as we can.